app/controllers/enrollments_controller.rb

```diff
@@ -268,6 +268,16 @@ def copies
            root: "enrollments"
   end
 
+  # GET enrollments/1/next_enrollment
+  def next_enrollment
+    @enrollments = policy_scope(Enrollment)
+      .where(previous_enrollment_id: params[:id])
+    render json: @enrollments,
+            each_serializer: LightEnrollmentSerializer,
+            adapter: :json,
+            root: "enrollments"
```

Comment:
:robot: lint robot says :
```suggestion
    render json: @enrollments,
           each_serializer: LightEnrollmentSerializer,
           adapter: :json,
           root: "enrollments"
```